packages/fluentui/docs/src/examples/components/Chat/Playground.tsx

```diff
@@ -126,17 +127,13 @@ const ChatPlayground: React.FunctionComponent = () => {
   };
 
   return (
-    <Chat
-      density={density}
-      items={[
-        {
-          gutter: <Avatar icon={<PersonIcon />} size={size} />,
-          key: '1',
-          message: <Chat.Message mine={mine} reactionGroupPosition={reactionGroupPosition} {...chatMessageProps} />,
-          ...chatItemProps,
-        },
-      ]}
-    />
+    <Chat density={density}>
+      <ChatItem
```

Comment:
Is this to prevent the playground from crashing locally? 😄

Reply:
Yes, and in production, too. This is because of bump React to 17 in our repo (production runs on React 16, that's why it works).